### PR TITLE
Replace Class.getClassLoader with io.netty.util.internal.PlatformDependent.getClassLoader in Openssl

### PR DESCRIPTION
**Pull request by skyguard1**
Motivation:

Replace Class.getClassLoader with io.netty.util.internal.PlatformDependent.getClassLoader in Openssl, please review this pr,thk

Modification:

Replace Class.getClassLoader with io.netty.util.internal.PlatformDependent.getClassLoader in Openssl

Result:

If there is no issue then describe the changes introduced by this PR.

### CODE REVIEW

**Comment by normanmaurer** [comment]
@netty-bot test this please

File: handler/src/main/java/io/netty/handler/ssl/OpenSsl.java [hunk 1]
```diff
@@ -120,7 +120,7 @@
         } else {
             // Test if netty-tcnative is in the classpath first.
             try {
-                Class.forName("io.netty.internal.tcnative.SSLContext", false, OpenSsl.class.getClassLoader());
+                Class.forName("io.netty.internal.tcnative.SSLContext", false, PlatformDependent.getClassLoader(OpenSsl.class));
```
Comment (amizurov) [inline handler/src/main/java/io/netty/handler/ssl/OpenSsl.java:123]: Fix check style please `Line is longer than 120 characters`

Comment (skyguard1) [reply handler/src/main/java/io/netty/handler/ssl/OpenSsl.java]: Thanks, i have done

**Comment by normanmaurer** [comment]
@netty-bot test this please

**Comment by normanmaurer** [comment]
@skyguard1 thanks a lot

**Comment by skyguard1** [comment]
> @skyguard1 thanks a lot
It is my pleasure,I will do more testing before submitting, appreciate it

**Pull request merged**